app/src/main/java/com/jerboa/ui/components/post/PostListing.kt

```diff
@@ -1443,6 +1445,19 @@ fun PostOptionsDialog(
                         icon = Icons.Outlined.Block,
                         onClick = onBlockCommunityClick,
                     )
+                    if (postView.post.thumbnail_url != null) {
+                        IconAndTextDrawerItem(
```

Comment:
This should probably be grouped with the other copy options in this menu, also allowing the creator to copy the thumbnail link.